narwhals/_spark_like/expr.py

```diff
@@ -557,6 +557,17 @@ def func(
 
         return self._with_window_function(func)
 
+    def fill_null(
+        self,
+        value: Any | None = None,
+        strategy: Literal["forward", "backward"] | None = None,
+        limit: int | None = None,
```

Comment:
```suggestion
        value: Any | None,
        strategy: Literal["forward", "backward"] | None,
        limit: int | None,
```
we generally avoid specifying default at the compliant level